xm-commons-security/src/main/java/com/icthh/xm/commons/security/oauth2/OAuth2JwtAccessTokenConverter.java

```diff
@@ -68,7 +68,9 @@ private boolean tryCreateSignatureVerifier() {
         try {
             SignatureVerifier verifier = signatureVerifierClient.getSignatureVerifier();
             if (verifier != null) {
-                setVerifier(verifier);
+                synchronized (signatureVerifierClient) {
```

Comment:
JEP 425 recommend to not use synchronized, better to use Lock

Reply:
Could you please explain? - just for clarification

Reply:
> JEP 425 recommend to not use synchronized, better to use Lock

Could you please explain? - just for clarification